charts/nri-k8s-operator/values.yaml

```diff
@@ -0,0 +1,98 @@
+# IMPORTANT: The Kubernetes cluster name
+# https://docs.newrelic.com/docs/kubernetes-monitoring-integration
+cluster: ""
+
+image:
+  repository: newrelic/k8s-metadata-injection
+  tag: ""  # Defaults to AppVersion
+  pullPolicy: IfNotPresent
+  ## It is possible to specify docker registry credentials
+  ## See https://kubernetes.io/docs/concepts/containers/images/#specifying-imagepullsecrets-on-a-pod
+  # pullSecrets:
+    # - name: regsecret
+
+admissionWebhooksPatchJob:
+  image:
+    repository: jettech/kube-webhook-certgen
+    tag: v1.5.0
+    pullPolicy: IfNotPresent
+  # Volume mounts to add to the job, you might want to mount tmp if Pod Security Policies
+  # Enforce a read-only root.
+  volumeMounts: []
+  #  - name: tmp
+  #    mountPath: /tmp
+
+  # Volumes to add to the job container
+  volumes: []
+  #  - name: tmp
+  #    emptyDir: {}
+
+replicas: 1
+
+resources:
+  limits:
+    memory: 80M
+  requests:
+    cpu: 100m
+    memory: 30M
+
+serviceAccount:
+  # Specifies whether a ServiceAccount should be created for the job and
+  # the deployment.
+  create: true
+  # The name of the ServiceAccount to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name:
+
+# Configure podSecurityContext
+podSecurityContext:
+  enabled: false
+  fsGroup: 1001
+  runAsUser: 1001
+  runAsGroup: 1001
+
+# Use cert manager for webhook certs, rather than the built-in patch job
+certManager:
+  enabled: false
+
+# Use custom tls certificates for the webhook, or let the chart handle it
+# TODO: Add support for defining custom certs for the webhook here
+# customTLSCertificate: {}
+
+# If you wish to provide additional annotations to apply to the pod(s), specify
+# them here
+# podAnnotations:
+
+# Pod scheduling priority
+# Ref: https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/
+# priorityClassName: high-priority
+
+# Webhook timeout
+#
+# Configure how long the API server should wait for a webhook to respond before treating the call as a failure
+#
+# Ref: https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#timeouts
+timeoutSeconds: 30
+
+# Operator configuration
+# The following are the default values for the operator
+config:
+  # sidecarInjection: {}  # Sidecar injection can be disabled by setting it to an empty object
+
+  sidecarInjection:  # List of sidecar injection policies, indexed by namespace
+    "*":  # This policy applies to all namespaces that do not have a specific entry in this map
```

Comment:
Would make sense to have it as a regex?

Reply:
If we add regex support, we'd need to convert this into a list to guarantee order of precedence. Might be a useful feature, but not sure if there will be demand for it. Might also be confusing for users, where `"custom"` would match `custom-namespace`.

Reply:
Another possible approach would be to put regex matchers between slashes, like `/[Rr]egex/`. We can definitely support this if we think it's valuable.

Reply:
Adding regexes increases complexity and makes following policies a bit less straightforward, so we decided to left it out for the time being.